### DISCUSSION TITLE
How to donate?

### DISCUSSION THREAD

**Discussion by ghost** [merbanan/rtl_433#2268] [General]
Hello, I like this project very much, all of you do a great job. Is there any way I can donate to this project to support your further work? Thank you!

**Comment by zuckschwerdt** [comment]
Thank you for the kind words and the generous offer.

We – I hope to speak for at least the maintainers and perhaps the regular contributors also – decline donations based on the belief that
financial goals often are not compatible with free software.

For one, prominent user-facing software gets attention, but we are all standing on (free) shoulders of giants here.
We also enjoy the lively contributions of many and couldn't distribute fairly.
And not least we all hack for the fun of it but wouldn't want to feel obliged to deliver.

There could be some wished-for feature where donations enable a developer to focus and devote time though.
Or a donation pool for developers to buy specific hardware maybe.

No concrete plans yet. We are open to suggestions and wishes anytime.

In the meantime, recommend rtl_433, help others get started, or just enjoy without obligations -- you are welcome.

**Comment by gdt** [comment]
I will observe to the OP that given a response like that, one can certainly donate to support Free Software in general, to someplace like FSF, FSFE, Software Freedom Conservancy, Codeberg, etc. 

Not relevant to rtl_433, but as an aside I will point out that  in another project where i am lead maintainer, I am contemplating leaving github for charitable hosting implemented with Free Software, and leaving googlegroups for a better mailinglist host, and it's possible we will end up with a fiscal sponsor that can accept donations to pay for expenses related to those services.

**Comment by merbanan** [comment]
In the beginning I selected github and google groups mainly to be able to scale. Regarding donations I think that we could accept them in a form of a donation in our name to a another organization/cause. Maybe Osmocom or other projects related to rtl_433.

**Comment by zuckschwerdt** [comment]
Yes, well put, cause and infrastructure should be carried by donations from the community. I wanted to convey that there is no need to pay developer time here, not even as encouragement (your offer alone is enough praise!) – we do enjoy hacking ;)
And by *we* I mean the spirit of the project as it appears to me. I don't mean to preempt anybody. If someone wants to start/do a funded feature-sprint or collect for interesting hardware that's great too.
